src/lib/yaraparse.py

```diff
@@ -476,11 +479,37 @@ def of_expr(self, condition: OfExpression) -> Optional[UrsaExpression]:
             return None
 
     def gt_expr(self, condition: GtExpression) -> Optional[UrsaExpression]:
-        left = self.traverse(condition.left_operand)
-        right = self.traverse(condition.right_operand)
-        return left or right
+        """ From `#a > 3` we can deduce, that #a must occur at least once """
+        return self.traverse(condition.left_operand)
```

Comment:
It seems that both `#a > 3` and `3 < #a` are legal conditions, will checking just the left one work?

Reply:
nvm